jenkins-scripts/dsl/sdformat.dsl

```diff
@@ -362,8 +364,8 @@ all_branches.each { branch ->
       }
   }
 
-// 2. master / @ SCM/Daily
-all_branches = sdformat_supported_branches + 'master'
+// 2. default / @ SCM/Daily
+all_branches = sdformat_supported_branches + 'default'
```

Comment:
revert this from `default` to `master`?

Reply:
ouch, thanks 97bc6ba91b36a12154091c36fdf00140d4d984c7